_rules/invalid-form-field-value-36b590.md

```diff
@@ -0,0 +1,241 @@
+
+```html
+<form>
+	<h2 id="error">
+		Name and color cannot be left unfilled. Please complete all required fields.
+	</h2>
+	<fieldset>
+		<legend>Your data</legend>
+		<label for="name">Name (required)</label>
+		<input type="text" id="name" required />
+		<br />
+		<label for="address">Address</label>
+		<input type="text" id="address" />
+	</fieldset>
+	<fieldset>
+		<legend>Pick a color (required)</legend>
+		<label><input type="radio" name="color" value="blue" required />Blue</label>
+		<label><input type="radio" name="color" value="yellow" />Yellow</label>
+	</fieldset>
+	<input type="button" value="Submit" aria-describedby="error" />
+</form>
+```
+
+#### Passed Example 3
+
+The `input` element does not have a [form field error indicator][].
+
+```html
+<form>
+	<label for="age">Age (years)</label>
+	<input type="number" id="age" />
+	<input type="button" value="Submit" />
+</form>
+```
+
+### Failed
+
+#### Failed Example 1
+
+The multiple `input` elements share a [form field error indicator][] but its message does not identify the elements that caused the error nor describes the cause of the error.
+
+```html
+<form>
+	<div id="error">Please fill the field correctly.</div>
+	<label for="age">Age (years)</label>
+	<input type="number" id="age" />
+	<label for="name">Name</label>
+	<input type="text" id="name" />
+	<input type="button" value="Submit" />
+</form>
+```
+
+#### Failed Example 2
+
+The `input` element has a [form field error indicator][] but its message does not describe the cause of the error.
+
+```html
+<form>
+	<label for="age">Age (years)</label>
+	<input type="number" id="age" />
+	<span id="error">Please enter a correct age.</span><br />
+	<input type="button" value="Submit" />
+</form>
+```
+
+#### Failed Example 3
+
+The `input` element has a [form field error indicator][] that identifies it (by referencing its label) and describes the cause of the error but the message is not [visible][].
+
+```html
+<form>
+	<label for="age">Age (years)</label>
+	<input type="number" id="age" value="0" />
+	<span id="error" style="display: none;">Invalid value for age. Age must be at least 1.</span><br />
+	<input type="button" value="Submit" aria-describedby="error" />
+</form>
+```
+
+#### Failed Example 4
+
+The `input` element has a [form field error indicator][] that identifies it (by referencing its label) and describes the cause of the error but the message is not [included in the accessibility tree][].
+
+```html
+<form>
+	<label for="age">Age (years)</label>
+	<input type="number" id="age" value="0" />
+	<span id="error" aria-hidden="true">Invalid value for age. Age must be at least 1.</span><br />
+	<input type="button" value="Submit" />
+</form>
+```
+
+#### Failed Example 5
+
+The multiple `input` elements share a [form field error indicator][]. The message describes the cause of the error but does not allow to identify the elements that caused the error because the same label is used in different `fieldset`s.
```

Comment:
```suggestion
These multiple `input` elements share a [form field error indicator][]. The message describes the cause of the error but does not allow to identify the elements that caused the error because the same label is used in different `fieldset`s.
```